<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QImageProcWidgetClass</class>
 <widget class="QWidget" name="QImageProcWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImageProc</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>图像线程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>传输线程</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLableModel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>545</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>处理模块</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelFrame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLableTransmit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelMsg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>311</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnState">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View     Camera</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelServer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelCoilID">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QImageProc.qrc"/>
 </resources>
 <connections/>
</ui>
